CONTRIBUTING.md

```diff
@@ -1,33 +1,5 @@
 # Contributing Guide
 
-## Running @cloudfour/eslint-plugin on itself
+## Release Process
 
-In order to run `npm run lint` on this project, it needs to have a symlink to itself in `node_modules`, so that ESLint can find it. To set up this symlink, run `npm link` and then `npm link @cloudfour/eslint-plugin`. Then you should be able to run `npm run lint`.
-
-## Pull Requests
-
-If you are submitting a pull request that includes changes that will affect places where this ESLint config is installed, add a Changeset file to describe the outward-facing changes:
```

Comment:
Changesets is nice, but a bit of a maintenance burden. I opted to rip it out and just use a standard release process like we do for our Stylelint config.